scripts/run-localnet-nodes.sh

```diff
@@ -0,0 +1,478 @@
+
+# Logging functions
+log_info() {
+    echo -e "${BLUE}[INFO]${NC} $1"
+}
+
+log_success() {
+    echo -e "${GREEN}[SUCCESS]${NC} $1"
+}
+
+log_warning() {
+    echo -e "${YELLOW}[WARNING]${NC} $1"
+}
+
+log_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+# Check if neo-cli exists
+check_neo_cli() {
+    log_info "Using NEO_CLI_DIR: $NEO_CLI_DIR"
+    if [ ! -f "$NEO_CLI_DIR/neo-cli" ] && [ ! -f "$NEO_CLI_DIR/neo-cli.dll" ]; then
+        log_error "neo-cli not found in $NEO_CLI_DIR"
+        log_info "Please build the project first: dotnet build"
+        log_info "Or set NEO_CLI_DIR environment variable to the correct path"
+        exit 1
+    fi
+}
+
+# An Array of addresses, NOTE: just for test
+ADDRESSES=(
+    "NSL83LVKbvCpg5gjWC9bfsmERN5kRJSs9d"
+    "NRPf2BLaP595UFybH1nwrExJSt5ZGbKnjd"
+    "NXRrR4VU3TJyZ6iPBfvoddRKAGynVPvjKm"
+    "NfGwaZPHGXLqZ17U7p5hqkZGivArXbXUbL"
+    "NjCgqnnJpCsRQwEayWy1cZSWVwQ7eRejRq"
+    "NYXoVMFa3ekGUnX4qzk8DTD2hhs5aSh2k4"
+    "NVBkHV59hTDxXAGas51eMtWhfxa6VDxf38"
+)
+
+# An Array of keys
+KEYS=(
+    "6PYVdEBZe7Mg4CiikuCXkEpcbwX7WXT72xfHTYd6hJzRWN3iBPDfGis7kV"
+    "6PYNZ7WDsjXwn2Mo8T3N7fTw7ZSfY71MXbVeRf1zZjv2baEdjbWNHm5mGQ"
+    "6PYXGkpWLLXtyC6cQthCcShioQJupRvyhDrz6xfLyiEa9HeJW4oTb4aJHP"
+    "6PYUCCNgCrVrB5vpCbsFwzEA7d2SkCzCTYMyhYw2TL51CaGeie2UWyehzw"
+    "6PYQpWR6CGrWDKauPWfVEfmwMKp2xKFod4X1AvV39ud5qhaSkrsFQeCBPy"
+    "6PYTm6sJLR1oWX2svdJkzWhkbqTAGurEybDdcCTBa19WNzDuFXURX2NAaE"
+    "6PYLnvLHYiVfxzKYMFtaZakWxkyj3WDX86PwBBYrYgkotBms6MWjfA2WHU"
+)
+
+# An Array of scripts
+SCRIPTS=(
+    "DCEChSZdyIWdBeHkKpDWwpqd4VUx6sGCSJdD5qlHgX0qn2ZBVuezJw=="
+    "DCECozKyXb9hGPwlv2Tw2DALu2I7eDRDcazwy1ByffMtnbNBVuezJw=="
+    "DCECqgIsK8NhTSOvwSFvxD2tkHINHilrTgm37izZvrgNm+pBVuezJw=="
+    "DCEDabXhB8SMjperdGnbbr8JAZz7MiPToYxK+iFwQoE9+d5BVuezJw=="
+    "DCECnkPTdNxK3KFYu0ZbSthBegdmQaU5UOPLccY0PdJYk9RBVuezJw=="
+    "DCEChLsd71mcGde7lMvdiOx+1IXbId6mTIa7kXYi+1ac6cpBVuezJw=="
+    "DCECwt/bG9EMosR8IoCV7biuh20TLuae8H4/+okS2TPWhq9BVuezJw=="
+)
+
+
+# Generate configuration for a specific node
+generate_node_config() {
+    local node_id=$1
+    local port=$((BASE_PORT + node_id))
+    local rpc_port=$((BASE_RPC_PORT + node_id))
+    local data_dir="$BASE_DATA_DIR/node_$node_id"
+    local config_file="$data_dir/config.json"
+    local wallet_file="$data_dir/wallet.json"
+
+    log_info "Generating config for node $node_id (port: $port, rpc: $rpc_port)"
+    
+    # Create data directory
+    mkdir -p "$data_dir"
+
+    # Generate seed list (all other nodes)
+    local seed_list=""
+    for i in $(seq 0 $((NODE_COUNT-1))); do
+        local seed_port=$((BASE_PORT + i))
+        if [ -n "$seed_list" ]; then
+            seed_list="$seed_list,"
+        fi
+        seed_list="$seed_list\"localhost:$seed_port\""
+    done
+    
+    # Create configuration file
+    cat > "$config_file" << EOF
+{
+  "ApplicationConfiguration": {
+    "Logger": {
+      "Path": "Logs",
+      "ConsoleOutput": false,
```

Comment:
```suggestion
      "ConsoleOutput": true,
```
Enable console ouput to ensure neo.log can get logs.